### ISSUE TITLE
Add recipients

### ISSUE THREAD

**Issue by Burton** [JoshData/cotaskme#5]
- navPostOutgoing accepts email addresses; null value = active user's email address
- Incoming list displays tasks assigned to active user's address
- Outgoing list displays all tasks generated by but not assigned to active user's address

**Comment by JoshData** [comment]
I was thinking a little about this. Imagine if everyone registered via Twitter (or Github). You could then @-name people to find them. @-names are usually easier to remember than email addresses (right?). Or we make people choose new handles. (This doesn't mean they couldn't also paste an email address.)

**Comment by Burton** [comment]
Absolutely, I think Twitter would be awesome. The @ syntax really makes sense for this context, and in general, I favor Twitter auth over other mechanisms (Google, Facebook) for privacy reasons. 

On a related note, do you think friending is a prerequisite for sending someone a task? By default, with no users to start with, I should be able to accept tasks from anyone, whether they have signed up for this service or not. Right?

**Comment by Burton** [comment]
Auto-validate/auto-suggest upon typing the To: username, a la Twitter.

This will pull Twitter usernames in our user database AND email addresses that the active user has tasked in the past.
